packages/amplify-graphql-index-transformer/src/__tests__/__snapshots__/amplify-graphql-index-transformer.test.ts.snap

```diff
@@ -1184,9 +1181,6 @@ $util.toJson({})
 #end
 #if( !$util.isNullOrEmpty($filter) )
   #set( $filterExpression = $util.parseJson($util.transform.toDynamoDBFilterExpression($filter)) )
-  #if( $util.isNullOrEmpty($filterExpression) )
-    $util.error(\\"Unable to process the filter expression\\", \\"Unrecognized Filter\\")
-  #end
```

Comment:
We don't want to revert this, do we?

Reply:
Good catch. This was removed in error during rebase. I've added it back.